feat: add master config option to provide custom logo (#9664)
determined-ai/determined: docs/release-notes/custom-logo.rst

:orphan:

**New Features**

> -  Add configuration options to the master config for specifying a custom logo to be displayed

Reviewer: ```suggestion
- Add a ``ui_customization`` option to the :ref:`master configuration <master-config-reference>` for specifying a custom logo for the WebUI.
```

> -  Add a set of configuration options to master config to allow providing a custom logo to be shown

Reviewer: ```suggestion
- Add configuration options to the master config for specifying a custom logo to be displayed
```

>       in the WebUI.

Reviewer: ```suggestion
```
